compiler/z/codegen/OpMemToMem.cpp

```diff
@@ -1636,6 +1635,8 @@ MemCmpVarLenSignMacroOp::generate(TR::Register* dstReg, TR::Register* srcReg, TR
    {
    TR::Instruction *cursorBefore = cursor;
    _resultReg = _cg->allocateRegister();
+   generateRRInstruction(_cg, TR::InstOpCode::getXORRegOpCode(), _rootNode, _resultReg, _resultReg);
```

Comment:
Why can't we use the subtract operation here and branch if result is zero ? This will save one extra instruction.